src/main/java/it/pagopa/transactions/configurations/AzureStorageConfig.java

```diff
@@ -168,6 +170,28 @@ public QueueAsyncClient transactionAuthorizationRequestedQueueAsyncClientV2(
         return buildQueueAsyncClient(storageConnectionString, queueName, jsonSerializerV2);
     }
 
+    @Bean("walletAsyncQueueClient")
+    @ConditionalOnExpression("${feature.walletusage.enabled:false}")
+    public WalletAsyncQueueClient walletUsageQueueAsyncClient(
+                                                              @Value(
+                                                                  "${azurestorage.wallet.connectionstring}"
+                                                              ) String storageConnectionString,
+                                                              @Value(
+                                                                  "${azurestorage.queues.walletusage.ttlSeconds}"
+                                                              ) int secondsTtl,
+                                                              @Value(
+                                                                  "${azurestorage.queues.walletusage.name}"
+                                                              ) String queueName,
+                                                              JsonSerializer jsonSerializerV2
```

Comment:
this json serializer v2 is the one from [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/6544e84907d7fb675b1ffa34a5d73e8a88356875/src/main/java/it/pagopa/transactions/configurations/AzureStorageConfig.java#L30-L34) that have mixin classes configured for eCommerce v2 events with type resolver configured for v2 events. maybe here an instance of `StrictJsonSerializerProvider().createInstance()` should be sufficient